luigi-tutorials/tutorial-container.md

```diff
@@ -0,0 +1,321 @@
+                 * Defines label of the Luigi component
+                 */
+                label: {
+                    type: "string",
+                    defaultValue: ""
+                },
+
+                /**
+                 * Defines the context to use
+                 */
+                context: {
+                    type: "object",
+                    defaultValue: {}
+                },
+
+                theme: {
+                    type: "string",
+                    defaultValue: ""
+                }
+            },
+            methods: ["updateContext", "sendCustomMessage", "closeAlert"]
+        }
+    });
+
+    LuigiContainer.prototype._mapIsWebComponent = function(vIsWebComponent) {
+        return vIsWebComponent !== "false" && vIsWebComponent ? "true" : undefined;
+    };
+
+    /**
+     * Updates the context of the Luigi component by sending a message to the iFrame / Web Component that sets the context
+     *
+     * @public
+     * @name sap.me.shared.LuigiContainer#updateContext
+     * @param contextObj the context data
+     * @param internal internal luigi legacy data
+     * @function
+    */
+
+    /**
+     * Send a custom message to the referenced iFrame or web component
+     *
+     * @public
+     * @name sap.me.shared.LuigiContainer#sendCustomMessage
+     * @param id the id of the event
+     * @param data data to be sent alongside the custom message
+     * @function
+     */
+
+    /**
+     * Send a message to the Luigi component notifying the alert has been closed
+     *
+     * @public
+     * @name sap.me.shared.LuigiContainer#closeAlert
+     * @param id the id of the alert being closed
+     * @param dismissKey the dismiss key being sent if any
+     * @function
+     */
+
+    return LuigiContainer;
+});
+```
+
+4. In command prompt/Terminal, download the Luigi Container npm package: 
+
+```shell
+npm i @luigi-project/container
+```
+
+5. Go to the `package-lock.json` file and ensure that the ` @luigi-project/container` and `ui5-tooling-modules` dependencies are added:
+
+```json
+[...]
+"name": "luigi.ui5app",
+"version": "1.0.0",
+"license": "Apache-2.0",
+  "dependencies": {
+// Around line 11: 
+    "@luigi-project/container": "^1.0.0-rc7"
+  },
+"devDependencies": {
+[...]
+// Around line 23: 
+"ui5-tooling-modules": "^3.0.8"
+}
+```
+
+### Use Luigi Container 
+
+1. In this step, you will use Luigi container in your app and configure the Luigi [viewURL](https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=viewurl) property in order to render a micro frontend on the page.
+
+Go to the `webapp/view/Main.view.xml` file of your UI5 application. Replace the content with the following: 
+
+```xml
+<mvc:View
+	controllerName="luigi.ui5app.controller.Main"
+	displayBlock="true"
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:core="sap.ui.core"
+	xmlns:luigi="luigi.ui5app.control"
+	core:require="{
+		formatter: 'luigi/ui5app/model/formatter',
```

Comment:
not used, can be removed

Reply:
Updated in [the latest commit.](https://github.com/alexandra-simeonova/drafts/pull/2/commits/a20e67a5eaea2606ccbe1c3ad68b7b34251fe099)